index.html

```diff
@@ -54,7 +54,174 @@ <h1>I’m Godfrey
+        </div>
   
+    </article>
+
+    <article id="pic3" class="work_content">
+      <img src="./image/Snapshoot Portfolio (3).svg" alt="">
+      <div class="work_description">
+        <h2>Multi-Post
+          Stories</h2>
+        <ul class="conduct">
+          <li class="conduct_head">CANOPY</li>
+          <li><img src="./image/Counter.svg" alt=""></li>
+          <li class="conduct_para">Back End Dev</li>
+          <li><img src="./image/Counter.svg" alt=""></li>
+          <li class="conduct_year">2015</li>
+        </ul>
+        <p class="work_paragraph">A daily selection of privately personalized reads; 
+          no accounts or sign-ups required.</p>
+        <ul class="work_languages">
+         <li>html</li>
+         <li>css</li>
+         <li>javascript</li>
+        </ul>
+          <button class="btn">See Project</button>
+      </div>
+
+  </article>
+
+  <article id="pic4" class="work_content">
+    <img src="./image/Snapshoot Portfolio (4).svg" alt="">
+    <div class="work_description">
+      <h2>Tonic</h2>
+      <ul class="conduct">
+        <li class="conduct_head">CANOPY</li>
+        <li><img src="./image/Counter.svg" alt=""></li>
+        <li class="conduct_para">Back End Dev</li>
+        <li><img src="./image/Counter.svg" alt=""></li>
+        <li class="conduct_year">2015</li>
+      </ul>
+      <p class="work_paragraph">A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+      <ul class="work_languages">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+      </ul>
+        <button class="btn">See Project</button>
+    </div>
+
+</article>
+    </div>
+</section>
+<section class="about">
+<article class="about_head">
+  <h2>About
+    Myself</h2>
+  <p>Hello I’m a software developer! 
+    I can help you build a product , 
+    feature or website Look through 
+    some of my work and experience! 
+    If you like what you see and have 
+    a project you need 
+    coded, don’t hestiate to contact me.</p>
+  <a href="">LET’S CONNECT</a>
+  <ul>
+    <li class="item1"><a href=""><img src="./image/Vectorgithub.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Linkedin iconlinkedin.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vectormedium.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vectortwitter (1).svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vector (3).svg" alt=""></a></li>
+  </ul>
+  <button class="btn">Get my resume</button>
+</article>
+<article class="about_language">
+<div class="item">
+  <h3>Languages</h3>
+  <img src="image/down.svg" alt="">
+</div>
+<ul class="about_lang">
+   <li>
+     <img src="image/javascript.svg" alt="">
+     <p>javascript</p>
+   </li>
+   <li>
+    <img src="image/html (2).svg" alt="">
+    <p>html</p>
+   </li>
+   <li>
+    <img src="image/css.svg" alt="">
+    <p>css</p>
+   </li>
```

Comment:
- [ ] I noticed that this section of your portfolio does not look nice on mobe screen sizes from `400px` and above, Please consider looking into this as your portfolio is meant to look exactly like the Figma design: 😄 👋 
- - Figma design:
![Screenshot 2023-03-31 012205](https://user-images.githubusercontent.com/107632780/228994847-a83556f8-9098-4be5-9751-06ea90c61a0f.png)

- - Your design:
![Screenshot 2023-03-31 012236](https://user-images.githubusercontent.com/107632780/228994887-3d28b9fb-422f-4033-b4a9-e43cc69dd16f.png)